curriculum/challenges/english/24-b1-english-for-developers/learn-common-phrasal-verbs-and-idioms/6848839f98b89790fd0874a2.md

```diff
@@ -0,0 +1,113 @@
+---
+id: 6848839f98b89790fd0874a2
+title: Task 13
+challengeType: 19
+dashedName: task-13
+---
+
+<!-- (Audio) Anna: I agree. We should also lay down some guidelines for communication during remote work. -->
+
+# --instructions--
+
+Listen to the audio and answer the question below.
+
+# --questions--
+
+## --text--
+
+What does Anna want to do about communication between people working remotely?
+
+## --answers--
+
+Start casual chats every morning.
+
+### --feedback--
+
+Anna wants something more formal than casual chats.
+
+---
+
+Send messages only when something urgent happens.
+
+### --feedback--
+
+Anna doesn't suggest limiting communication to urgent messages.
+
+---
+
+Hold long weekly meetings to improve connection.
+
+### --feedback--
+
+Anna doesn't mention meetings.
+
+---
+
+Create clear rules for how people should communicate.
+
+## --video-solution--
+
+4
+
+# --explanation--
+
+`To lay down`, in this context, means to create or establish official rules or procedures. For example:
```

Comment:
If `lay down` is a new phrase, I'd suggest using a type 22 task to teach it first.

Reply:
Let us test how this goes with a type-19. I'll make the change in an upcoming PR.